From d9d473e4486107a504e8ac68bd2ff4df26670a13 Mon Sep 17 00:00:00 2001
From: Ben Sartor <ben@simlar.org>
Date: Tue, 29 Nov 2016 00:07:04 +0100
Subject: [PATCH 2/2] zrtp now prefers AES256 and HS80

---
 coreapi/misc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/coreapi/misc.c b/coreapi/misc.c
index 125d632..a960d63 100644
--- a/coreapi/misc.c
+++ b/coreapi/misc.c
@@ -1451,7 +1451,7 @@ void linphone_core_set_tone(LinphoneCore *lc, LinphoneToneID id, const char *aud
 }
 
 const MSCryptoSuite * linphone_core_get_srtp_crypto_suites(LinphoneCore *lc){
-	const char *config= lp_config_get_string(lc->config, "sip", "srtp_crypto_suites", "AES_CM_128_HMAC_SHA1_80, AES_CM_128_HMAC_SHA1_32, AES_256_CM_HMAC_SHA1_80, AES_256_CM_HMAC_SHA1_32");
+	const char *config= lp_config_get_string(lc->config, "sip", "srtp_crypto_suites", "AES_256_CM_HMAC_SHA1_80, AES_256_CM_HMAC_SHA1_32, AES_CM_128_HMAC_SHA1_80, AES_CM_128_HMAC_SHA1_32");
 	char *tmp=ms_strdup(config);
 
 	char *sep;
-- 
2.1.4

