From 46e3233029c4d314829c488bf8cb4d7faf8410f2 Mon Sep 17 00:00:00 2001
From: Ben Sartor <ben@simlar.org>
Date: Thu, 5 Feb 2015 05:55:49 +0100
Subject: [PATCH 2/2] zrtp now prefers AES256 and HS80

---
 coreapi/misc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/coreapi/misc.c b/coreapi/misc.c
index 08c0699..c9f80e8 100644
--- a/coreapi/misc.c
+++ b/coreapi/misc.c
@@ -1533,7 +1533,7 @@ void linphone_core_set_tone(LinphoneCore *lc, LinphoneToneID id, const char *aud
 }
 
 const MSCryptoSuite * linphone_core_get_srtp_crypto_suites(LinphoneCore *lc){
-	const char *config=lp_config_get_string(lc->config,"sip","srtp_crypto_suites","AES_CM_128_HMAC_SHA1_80, AES_CM_128_HMAC_SHA1_32, AES_CM_256_HMAC_SHA1_80, AES_CM_256_HMAC_SHA1_32");
+	const char *config=lp_config_get_string(lc->config,"sip","srtp_crypto_suites","AES_CM_256_HMAC_SHA1_80, AES_CM_256_HMAC_SHA1_32, AES_CM_128_HMAC_SHA1_80, AES_CM_128_HMAC_SHA1_32");
 	char *tmp=ms_strdup(config);
 	char *sep;
 	char *pos;
-- 
2.1.4

