From 77c18cc2c44d567a75ced6fd7921d54932646a83 Mon Sep 17 00:00:00 2001
From: Erwan Croze <erwan.croze@belledonne-communications.com>
Date: Tue, 13 Mar 2018 10:20:31 +0100
Subject: [PATCH 1/2] Changing Android Support V4 version

---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index 8df9233c..b419dd66 100644
--- a/build.gradle
+++ b/build.gradle
@@ -48,7 +48,7 @@ dependencies {
         compile 'com.google.firebase:firebase-messaging:11.+'
     } else {
         compile fileTree(include: 'gcm.jar', dir: 'libs')
-        compile 'com.android.support:support-v4:+'
+        compile 'com.android.support:support-v4:27.1.0'
     }
 }
 
-- 
2.11.0

