From 1e8776378e7bc76a9877360ecd48069037ee0935 Mon Sep 17 00:00:00 2001
From: Ben Sartor <ben@simlar.org>
Date: Sat, 27 Oct 2018 20:54:51 +0200
Subject: [PATCH] reorder gradle repositories in order to fix build

---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index cc77667a..6f6eb83f 100644
--- a/build.gradle
+++ b/build.gradle
@@ -28,10 +28,10 @@ buildscript {
 
 allprojects {
     repositories {
+        google()
         jcenter()
         mavenCentral()
         mavenLocal()
-        google()
     }
 }
 
-- 
2.11.0

