From cbdceced474c7c1756e55e3d1fd2d206e3726e21 Mon Sep 17 00:00:00 2001
From: Ben Sartor <mail@ben-sartor.de>
Date: Fri, 5 Sep 2014 00:05:19 +0200
Subject: [PATCH] set android sdk version to 20

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index acb0140..99ffa83 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ LIBLINPHONE_VERSION=$(shell cd submodules/linphone && git describe --always)
 LINPHONE_ANDROID_DEBUG_VERSION=$(shell git describe --always)
 BELLESIP_VERSION_SCRIPT:=cat submodules/belle-sip/configure.ac | grep "AC_INIT(" | sed -e "s/.*belle-sip\]//" | sed -e "s/].*//" | sed -e "s/.*\[//"
 BELLESIP_VERSION=$(shell $(BELLESIP_VERSION_SCRIPT))
-ANDROID_MOST_RECENT_TARGET=$(shell android list target -c | grep android | tail -n1)
+ANDROID_MOST_RECENT_TARGET=android-20
 SQLITE_VERSION=3071700
 SQLITE_BASENAME=sqlite-amalgamation-$(SQLITE_VERSION)
 SQLITE_URL=http://www.sqlite.org/2013/$(SQLITE_BASENAME).zip
-- 
2.1.0

