From 71d2ac7d03dc2a89ad13c23f005cd3a2a12634c7 Mon Sep 17 00:00:00 2001
From: Ben Sartor <ben@simlar.org>
Date: Sun, 21 Apr 2019 15:08:12 +0200
Subject: [PATCH] fix build

---
 submodules/mediastreamer2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/submodules/mediastreamer2 b/submodules/mediastreamer2
index a64b5965..c54e7d15 160000
--- a/submodules/mediastreamer2
+++ b/submodules/mediastreamer2
@@ -1 +1 @@
-Subproject commit a64b59653b09852490acb3b642e4435d989097db
+Subproject commit c54e7d15df4485f21ce9419fd95bbfc569f4d665
-- 
2.20.1

