From b80661eaa1400761dcff49294869ce13ca78e8c0 Mon Sep 17 00:00:00 2001
From: Sylvain Berfini <sylvain.berfini@linphone.org>
Date: Wed, 10 Dec 2014 10:10:37 +0100
Subject: [PATCH 2/3] Updated libilbc

---
 submodules/libilbc-rfc3951 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/submodules/libilbc-rfc3951 b/submodules/libilbc-rfc3951
index d0b51b1..b0664ab 160000
--- a/submodules/libilbc-rfc3951
+++ b/submodules/libilbc-rfc3951
@@ -1 +1 @@
-Subproject commit d0b51b1a8604f726bb3a3b2609cdfbd7d4e43828
+Subproject commit b0664ab54875d9af22c100e8cb8d0155a83a5d36
-- 
2.1.4

