From cdf73a5757a43b64f4cb644950364bdf7735a00c Mon Sep 17 00:00:00 2001
From: Sylvain Berfini <sylvain.berfini@linphone.org>
Date: Wed, 10 Dec 2014 11:15:21 +0100
Subject: [PATCH 3/3] Updated libilbc to fix compil

---
 submodules/libilbc-rfc3951 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/submodules/libilbc-rfc3951 b/submodules/libilbc-rfc3951
index b0664ab..959cdce 160000
--- a/submodules/libilbc-rfc3951
+++ b/submodules/libilbc-rfc3951
@@ -1 +1 @@
-Subproject commit b0664ab54875d9af22c100e8cb8d0155a83a5d36
+Subproject commit 959cdce5ae41714969d2e7463bd6e2fd3d821ca5
-- 
2.1.4

